index.css

```diff
@@ -152,6 +163,207 @@ li {
   align-items: center;
 }
 
+.textside p {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills li {
+  background-color: #ebebff;
+  padding: 6px 5px;
+  border-radius: 8px;
+  font-size: 15px;
+  color: #396df2;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.about-skills li {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  width: 140px;
+}
+
+.card-ul li {
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.card-ul li:nth-child(1) {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills {
+  text-align: center;
+  display: flex;
+}
+
+.see-project {
+  background-color: #fff;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 9px;
+  padding: 5px 8px;
+  align-self: flex-start;
+  letter-spacing: 0.03em;
+  width: 140px;
+  height: 50px;
+  font-size: 15px;
+  font-weight: 700;
+}
+
+.see-project:hover {
+  background: #6070ff;
+  box-shadow: #4053fc3d;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.see-project:active {
+  background: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.see-project:disabled {
+  color: #5e6c84;
+  background-color: #fff;
+}
+
+.about-social {
+  gap: 22px;
+}
+
+#about {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  background-color: #fff;
+  padding: 2em 2em;
+  gap: 10px;
+  border-radius: 20px;
+}
```

Comment:
- [ ] Great work making adjustments to the vertical alignment of the **_social media_** icons and the **_Get My Resume_** button. However, you have not adjusted the general layout of the `About Myself` section. I would recommend that you use the CSS style declaration given by the former reviewer; `grid-template-columns: 40% 60%`. The _**Language**_ side of the `About Myself` section takes up more space than the other side. The screenshot below shows the way your design would be if you adopt the CSS declaration recommended 😎 :

![image](https://user-images.githubusercontent.com/77106962/220733721-8f9f73b3-f011-4c39-8a20-d21e46ced5af.png)

#### If you compare it to the current design that you have, you will notice that the _**Language**_ section occupies more space:

### Your Current Design

![image](https://user-images.githubusercontent.com/77106962/220734025-b4d1bf6d-bc3d-4fc3-bbbe-d5be545446cf.png)

Reply:
grid-template-column has been adjust to the correct ratio